src/providers/Asset.tsx

```diff
@@ -52,6 +53,7 @@ function AssetProvider({
   const [price, setPrice] = useState<BestPrice>()
   const [owner, setOwner] = useState<string>()
   const [error, setError] = useState<string>()
+  const [type, setType] = useState<string>()
```

Comment:
now that we are more specific in the type typing, this state should use the same typings then, hence my suggestion of using `MetadataMain['type']` from ocean.js also in the above `AssetProviderValue` so we do not have to repeat those hardcoded strings